parsl/executors/high_throughput/executor.py

```diff
@@ -421,15 +422,31 @@ def hold_worker(self, worker_id):
     @property
     def outstanding(self):
         outstanding_c = self.command_client.run("OUTSTANDING_C")
-        logger.debug("Got outstanding count: {}".format(outstanding_c))
+        # logger.debug("Got outstanding count: {}".format(outstanding_c))
         return outstanding_c
 
     @property
     def connected_workers(self):
         workers = self.command_client.run("MANAGERS")
-        logger.debug("Got managers: {}".format(workers))
+        # logger.debug("Got managers: {}".format(workers))
         return workers
 
+    def _hold_block(self, block_id):
+        """ Sends hold command to all managers which are in a specific block
```

Comment:
Will there be multiple managers in the same block? In my head there was a 1:1 correspondence.

Reply:
A block can have many nodes. Each node has 1 manager.